internal/dinosaur/pkg/handlers/admin_dinosaur.go

```diff
@@ -44,6 +50,14 @@ type AdminCentralHandler interface {
 	// a tenant. In particular, avoid two Central CRs appearing in the same
 	// tenant namespace. This may cause conflicts due to mixed resource ownership.
 	PatchName(w http.ResponseWriter, r *http.Request)
+	// GetCentralTrait tells wheter a central has the trait
+	GetCentralTrait(w http.ResponseWriter, r *http.Request)
+	// GetCentralTraits returns all central traits
+	GetCentralTraits(w http.ResponseWriter, r *http.Request)
```

Comment:
```suggestion
	ListCentralTraits(w http.ResponseWriter, r *http.Request)
```
To avoid confusion with `GetCentralTrait`.